test/models/maintenance_tasks/run_test.rb

```diff
@@ -164,6 +164,25 @@ class RunTest < ActiveSupport::TestCase
       run.persist_error(error)
     end
 
+    test "#persist_error does not raise on longer error class names" do
+      run = Run.create!(task_name: "Maintenance::ErrorTask")
+      error = ArgumentError.new("Something went wrong")
+      error.stubs(class: "SomeVeryLongErrorClassName #{"." * 20000}")
+      error.set_backtrace(["lib/foo.rb:42:in `bar'"])
+      assert_nothing_raised do
+        run.persist_error(error)
+      end
+    end
+
+    test "#persist_error does not raise on longer errors messages" do
+      run = Run.create!(task_name: "Maintenance::ErrorTask")
+      error = ArgumentError.new("SomeVeryLongErrorMessage #{"." * 20000}")
+      error.set_backtrace(["lib/foo.rb:42:in `bar'"])
+      assert_nothing_raised do
+        run.persist_error(error)
+      end
```

Comment:
That would depend on the database in use and this test is database agnostic. Currently, CI is running with `sqlite` therefore there's no limit. I ran this test using `mysql` and without the `truncate(attribute_name, value)` it would fail.

@etiennebarrie will bring `mysql` into the CI and then we can run the tests for each supported database.